docs/wiki/The CI environment - Deployment removal.md

```diff
@@ -0,0 +1,37 @@
+This pipeline and its scripts can be used to limit the number of deployments that are listed in Azure at any given time. This is especially useful for a scope like 'Management Group' is Azure does not provide any [automatic cleanup](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deployment-history-deletions?tabs=azure-powershell).
```

Comment:
Should this be added to any TOC? maybe the main CI environment page?

Reply:
Added